scripts/pyjoke

```diff
@@ -39,7 +39,14 @@ def main():
         parser.error(str(exc.message))
         exit(-1)
 
-    joke = get_joke(language=args.language, category=args.category)
+    try:
+        joke = pyjokes.get_joke(language=args.language, category=args.category)
+    except pyjokes.LanguageNotFoundError:
+        print('No such language %s' % args.language)
```

Comment:
I guess this error message would make more sense: `'No jokes found in language {}'.format(args.language)`.
I mean, if the language is a existing one you can't say there is no such a language, right?
You could also invite people to add jokes in the language.

P.S.: I don't care if you use the `format` function or the `%`.